platform-core-module/src/main/java/com/softicar/platform/core/module/transaction/data/TransactionDataDeleter.java

```diff
@@ -0,0 +1,81 @@
+package com.softicar.platform.core.module.transaction.data;
+
+import com.softicar.platform.common.core.exceptions.SofticarUserException;
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.module.IModule;
+import com.softicar.platform.db.runtime.table.IDbTable;
+import com.softicar.platform.db.runtime.table.dependency.DbTableDependencyGraph;
+import com.softicar.platform.emf.module.registry.CurrentEmfModuleRegistry;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class TransactionDataDeleter {
+
+	public void execute() {
+
+		for (IDbTable<?, ?> table: getTableList()) {
```

Comment:
We could extract the content of this loop to a private method, to improve legibility.

Reply:
I do not see much value in it, but did so